index.html

```diff
@@ -431,6 +431,35 @@ <h2>
             <hr>
             <!-- End template -->
 
+            <!-- Mission Support System  Start Template -->
+            <div class="pure-u-1 pure-u-md-1-1 pure-u-lg-1-1">
+                <div class="pure-g">
+                    <div class="pure-u-1 pure-u-md-1-2 logo">
+                        <img src="https://mss.readthedocs.io/en/stable/_static/mss-logo.png" />
+                    </div>
+                    <div class="pure-u-1 pure-u-md-1-2">
+                        <h2>
+                            MSS  - Mission Support System
+                        </h2>
+                    </div>
+                </div>
+                <br />
+                MSS is a web service based client/server application to plan atmospheric research flights. <br />
+                <p>
+                The Mission Support System (MSS) is a software that is written by scientists in the field of atmospheric science. The purpose is to have a tool that simplifies the process for planning a scientific flight in which parameters of the atmosphere are measured. The research aircrafts typically carry a comprehensive scientific payload comprised of data aquisition instruments by different companies and research institutions. The measurement of relevant parameters, for example the chemical composition of trace gases, temperature or aerosol particle characteristics, are needed to improve the scientific understanding of the processes in the atmosphere. This is of significant importance for the understanding for example of climate change or the recovery of the ozone hole. </p>
+                <div class="pure-g">
+                  <div class="pure-g" style="text-align: center;">
+￼						<div class="pure-u-1 pure-u-md-1-4"><p><a href="https://bitbucket.org/wxmetvis/mss/wiki/GSOC2020/Website">Website</a></p></div>
```

Comment:
This link seems to be broken.  What should it be?

Reply:
I redo this, not sure why the merge looks like this